schedule/__init__.py

```diff
@@ -341,16 +341,29 @@ def at(self, time_str):
         :param time_str: A string in `XX:YY` format.
         :return: The invoked job instance
         """
-        assert self.unit in ('days', 'hours') or self.start_day
-        hour, minute = time_str.split(':')
-        minute = int(minute)
+        assert self.unit in ('days', 'hours', 'minutes') or self.start_day
+        if time_str.count(':') == 2:
+            hour, minute, second = time_str.split(':')
+        elif time_str.count(':') == 1 and self.unit == 'minutes':
```

Comment:
I'm wondering if the parser stuff is easier to read if we just do a `.split(':')` at the start and then count the number of items to explode the list into h/m/s?

Reply:
I made a change which I think does what you're asking for here, let me know if you envisioned it working another way.